addons/l10n_in/views/account_invoice_views.xml

```diff
@@ -11,6 +11,11 @@
                         <field name="l10n_in_hsn_code_warning" widget="actionable_errors"/>
                     </div>
                 </div>
+                <field name="l10n_in_display_null_pan_warning" invisible="1"/>
+                <div class="alert alert-warning main mb-0" role="alert"
+                        invisible="not l10n_in_display_null_pan_warning">
+                        If the partner has no PAN Number, please apply TDS / TCS at a higher rate.
+                </div>
```

Comment:
nitpick: indentation should be 4 space instead of 8. To make it similar with the other fields below
```suggestion
                <div class="alert alert-warning main mb-0" role="alert"
                    invisible="not l10n_in_display_null_pan_warning">
                    If the partner has no PAN Number, please apply TDS / TCS at a higher rate.